packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1129,6 +1208,24 @@ export class UmbrellaStore<M extends BaseMetadata> {
       }
     );
 
+    const channelNotificationSettings: LoadableResource<ChannelsNotificationSettingsAsyncResult> =
+      {
+        signal: DerivedSignal.from(
+          (): ChannelsNotificationSettingsAsyncResult => {
+            const asyncResult = this.#channelsNotificationSettings.get();
+            if (asyncResult.isLoading || asyncResult.error) {
+              return asyncResult;
+            }
+
+            return {
+              isLoading: false,
+              settings: nn(this.channelNotificationSettings.signal.get()),
+            };
+          }
+        ),
```

Comment:
Subtle, but this probably needs a `shallow` equality function here:

```suggestion
          },
          shallow,
        ),
```